composites/Plugin/SnippetEditor/components/SnippetEditorFields.js

```diff
@@ -114,34 +117,55 @@ const SimulatedLabel = styled.div`
 	cursor: pointer;
 	font-size: 16px;
 	font-family: Arial, Roboto-Regular, HelveticaNeue, sans-serif;
+	margin-bottom: 5px;
+`;
+
+const TriggerReplacementVariableSuggestionsButton = Button.extend`
+	height: 33px;
+	border: 1px solid #dbdbdb;
+	box-shadow: none;
+	font-family: Arial, Roboto-Regular, HelveticaNeue, sans-serif;
+	fill: ${ colors.$color_grey_dark };
+	padding-left: 8px;
+	float: right;
+	margin-top: -35px; // negative height + 2 for spacing
+
+	${ props => props.isSmallerThanMobileWidth && `
```

Comment:
this should be `hasMobileWidth` (which is the passed prop) while `isSmallerThanMobileWidth` is the state property the passed prop depends on. Currently, when the viewport is less than the mobile width (356), there's no change in the CSS. I'd consider to rename `hasMobileWidth` to something more appropriate.

<img width="353" alt="screen shot 2018-06-07 at 15 06 18" src="https://user-images.githubusercontent.com/1682452/41101703-2dc50fb2-6a65-11e8-9f05-666f7c57eb6f.png">

Reply:
Forgot this in my rename to `isSmallerThanMobileWidth`, will fix. Good spot!